README.md

```diff
@@ -38,7 +44,7 @@ jobs:
       
       
     - name: Run EC Validator
-      uses: ./
+      uses: enterprise-contract/actions/verify@v1
```

Comment:
You're planning to rename the repo I take it?

Reply:
@simonbaird Yes, I looked up version tagging, and it is not fully supported for actions. I think it will cause more of a headache over time if we have one repo with all the actions.